docs/docusaurus/docs/guides/validation/checkpoints/how_to_configure_a_new_checkpoint_using_test_yaml_config.md

```diff
@@ -4,13 +4,9 @@ title: How to configure a new Checkpoint using test_yaml_config
 import Prerequsities from '../../connecting_to_your_data/components/prerequisites.jsx'
 import TechnicalTag from '@site/docs/term_tags/_tag.mdx';
 
-import InProgress from '/docs/components/warnings/_in_progress.md'
+This how-to guide demonstrates advanced examples for configuring a <TechnicalTag tag="checkpoint" text="Checkpoint" /> using `test_yaml_config`. **Note:** For a basic guide on creating a new Checkpoint, please see [How to create a new Checkpoint](../../../guides/validation/checkpoints/how_to_create_a_new_checkpoint.md).
 
-<InProgress />
-
-This how-to guide demonstrates advanced examples for configuring a <TechnicalTag tag="checkpoint" text="Checkpoint" /> using ``test_yaml_config``. **Note:** For a basic guide on creating a new Checkpoint, please see [How to create a new Checkpoint](../../../guides/validation/checkpoints/how_to_create_a_new_checkpoint.md).
-
-``test_yaml_config`` is a convenience method for configuring the moving parts of a Great Expectations deployment. It allows you to quickly test out configs for <TechnicalTag tag="datasource" text="Datasources" />, <TechnicalTag tag="store" text="Stores" />, and Checkpoints. ``test_yaml_config`` is primarily intended for use within a notebook, where you can iterate through an edit-run-check loop in seconds.
+`test_yaml_config` is a convenience method for configuring the moving parts of a Great Expectations deployment. It allows you to quickly test out configs for <TechnicalTag tag="datasource" text="Datasources" />, <TechnicalTag tag="store" text="Stores" />, and Checkpoints. `test_yaml_config` is primarily intended for use within a notebook, where you can iterate through an edit-run-check loop in seconds.
```

Comment:
```suggestion
`test_yaml_config` lets you configure and test the components of a Great Expectations deployment, including <TechnicalTag tag="datasource" text="Datasources" />, <TechnicalTag tag="store" text="Stores" />, and Checkpoints. `test_yaml_config` is intended for use within a Jupyter or similar notebook, where you can use an edit-run-check loop to quickly test your changes before implementation.
```
If you don't implement the suggested edit, change _convenience_ to _convenient_.